### PR TITLE
Add 'MDADM_STATE' custom component

### PR DESCRIPTION
**Pull request by LorenzoVasi**
:-)

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @LorenzoVasi, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#392).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/mdadm_state [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "RAID Status - MDADM",
+  "owner": ["@LorenzoVasi"],
+  "manifest": "https://github.com/LorenzoVasi/HA_mdadm/blob/main/custom_components/mdadm_state/manifest.json",
```
Comment (frenck) [inline components/mdadm_state:4]: This needs to be the raw URL

```suggestion
  "manifest": "https://raw.githubusercontent.com/LorenzoVasi/HA_mdadm/main/custom_components/mdadm_state/manifest.json",
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @LorenzoVasi 👍

**Comment by LorenzoVasi** [comment]
Thanks to you, @frenck <3

**Comment by LorenzoVasi** [comment]
One fast question: 
after that, what should I do for add the component into HACS store?
Thanks again 👍

**Comment by frenck** [comment]
> after that, what should I do for add the component into HACS store?

That question is better asked @ HACS. That project is not maintained by the Home Assistant project.

**Comment by LorenzoVasi** [comment]
Okok thankssss :)

**Pull request merged**